ek9000App/src/scripts/ci_check.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+
+import json
+import argparse
+import jsonschema
```

Comment:
We probably should add a `requirements.txt` for these scripts somewhere.

I guess we just got lucky that GHA has jsonschema/lxml installed in its default Python environment? I wouldn't have expected it to have anything but the basics, so that surprised me a bit. In our typical CI pipelines, we use https://github.com/actions/setup-python for pip-based testing.

Reply:
Probably a good idea, totally slipped my mind!!